java-plugin/src/main/java/fr/greencodeinitiative/java/checks/AvoidSpringRepositoryCallInLoopWithStream.java

```diff
@@ -0,0 +1,65 @@
+package fr.greencodeinitiative.java.checks;
+
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+import org.sonar.plugins.java.api.IssuableSubscriptionVisitor;
+import org.sonar.plugins.java.api.semantic.MethodMatchers;
+import org.sonar.plugins.java.api.tree.*;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Rule(
+        key = "EC_CRJVM208",
```

Comment:
please add a new line with this id in `RULES.md` file (just next to the one talking about spring repository in loops)
please add a new entry in `CHANGELOG.md` file